addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler/FormHandlerParam.java

```diff
@@ -56,16 +56,34 @@ public class FormHandlerParam extends VersionedAbstractParam {
     private static final String CONFIRM_REMOVE_TOKEN_KEY =
             FORM_HANDLER_BASE_KEY + ".confirmRemoveField";
 
-    private static final Map<String, String> DEFAULT_KEY_VALUE_PAIRS = new HashMap<>();
-
-    static {
-        DEFAULT_KEY_VALUE_PAIRS.put("color", "#ffffff");
-        DEFAULT_KEY_VALUE_PAIRS.put("email", "foo-bar@example.com");
-        DEFAULT_KEY_VALUE_PAIRS.put("name", "ZAP");
-        DEFAULT_KEY_VALUE_PAIRS.put("password", "ZAP");
-        DEFAULT_KEY_VALUE_PAIRS.put("phone", "9999999999");
-        DEFAULT_KEY_VALUE_PAIRS.put("url", "https://www.example.com");
-    }
+    private static final List<FormHandlerParamField> DEFAULT_FIELDS =
+            List.of(
+                    new FormHandlerParamField("color", "#ffffff"),
+                    new FormHandlerParamField("email", "foo-bar@example.com"),
+                    new FormHandlerParamField("name", "ZAP"),
+                    new FormHandlerParamField("password", "ZAP"),
+                    new FormHandlerParamField("phone", "9999999999"),
+                    new FormHandlerParamField("url", "https://www.example.com"),
+                    new FormHandlerParamField(
+                            "(?i)_?back[-_]?(?:link|uri|url)?",
+                            "https://www.example.com",
+                            true,
+                            true),
+                    new FormHandlerParamField("(?i)_?bg[-_]?colou?r", "#FFFFFF", true, true),
```

Comment:
Could we update this so that this matches "color" as well? Then we can remove the "color" entry.
Same comment for the url regex above.

Reply:
Well that gets us back to the what if they're removed problem.